internal/pkg/config/service.go

```diff
@@ -12,6 +12,19 @@ import (
 	"github.com/aws/aws-sdk-go/service/ssm"
 )
 
+const (
```

Comment:
I'd love for us to eventually move this to the template pkg instead of config. Not needed in this PR just sharing my thoughts :)